### ISSUE TITLE
Primary CTA should auto respond to theme context

### ISSUE THREAD

**Issue by starryeyez024** [patternfly/patternfly-elements#641]
![image](https://user-images.githubusercontent.com/456863/69666586-7ab4ed00-105a-11ea-88ff-315a9be12f01.png)

Primary CTA needs to opt into the theme context, i.e. if placed on a `<pfe-card pfe-color="accent">` this invokes the `--theme:saturated`. 

The `<pfe-cta pfe-priority=primary>` should be listening for broadcast variable and should change to the white variation when on a saturated theme.

**Comment by castastrophe** [comment]
This PR should address this: https://github.com/patternfly/patternfly-elements/pull/630
